app/build.gradle

```diff
@@ -37,6 +37,7 @@ android {
         // buildConfigField 'String', 'MAIL_API_PREPROD', '"https://mail.preprod.dev.infomaniak.ch"'
         buildConfigField 'String', 'SHOP_URL', '"https://ik.me"'
         buildConfigField 'String', 'CHATBOT_URL', '"https://www.infomaniak.com/chatbot"'
+        buildConfigField 'String', 'MANAGE_SIGNATURES_URL', '"https://mail.infomaniak.com/0/settings/signatures"'
```

Comment:
Do we create a whole variable for it? Or do we create an ApiRoute that depends on `MAIL_API`?

Reply:
@Infomaniak/android what do you all think?

Reply:
We already have `'MAIL_API', '"https://mail.infomaniak.com"'`.
Re-use it, like the other routes are doing.

Reply:
Keep it this way.